libraries/adb-scrcpy/CHANGELOG.md

```
@@ -1,5 +1,33 @@
+### Patch Changes
+
+- 4a228d1: Fix default version for client version 3.3.2
+
+## 2.2.0
+
+### Minor Changes
+
+- f34724f: Add support for Scrcpy server version 3.3.2 (no protocol changes)
+
+### Patch Changes
+
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)